src/main/resources/application.properties

```diff
@@ -1,26 +1,34 @@
-configuration.cache.expire=120
-vendor.id.cache.expire=600
-vendor.cache.expire=86400
-package.cache.expire=86400
-resource.cache.expire=86400
-title.cache.expire=86400
-configuration.base.url.default=https://sandbox.ebsco.io
-holdings.status.check.delay=300000
-holdings.status.retry.count=20
-holdings.report.status.check.delay=120000
-holdings.report.status.retry.count=30
-holdings.status.audit.expiration.period=2592000000
+access.types.default.number.limit.value=15
+
+db.exception.translator.name=postgresql
+
+# Possible values are TransactionLoadServiceFacade, DefaultLoadServiceFacade
+# Qualifier of a bean that implements LoadServiceFacade interface.
+holdings.load.implementation.qualifier=DefaultLoadServiceFacade
+holdings.load.retry.count=3
+holdings.load.retry.delay=10800000
 holdings.page.retry.count=3
 holdings.page.retry.delay=900000
+holdings.report.status.check.delay=120000
+holdings.report.status.retry.count=30
+holdings.snapshot.refresh.period=86400000
 holdings.snapshot.retry.count=3
 holdings.snapshot.retry.delay=10800000
-holdings.snapshot.refresh.period=86400000
-holdings.load.retry.delay=10800000
-holdings.load.retry.count=3
+holdings.status.audit.expiration.period=2592000000
+holdings.status.check.delay=300000
+holdings.status.retry.count=20
 holdings.timeout=3600000
-# Qualifier of a bean that implements LoadServiceFacade interface.
-# Possible values are TransactionLoadServiceFacade, DefaultLoadServiceFacade
-holdings.load.implementation.qualifier=DefaultLoadServiceFacade
-access.types.default.number.limit.value=15
-db.exception.translator.name=postgresql
+
+# Credentials properties
+kb.ebsco.credentials.name.length.max=255
+kb.ebsco.credentials.url.default=https://sandbox.ebsco.io
```

Comment:
My comment regarding using https://api.ebsco.com -- applies to these properties instead of test properties

Reply:
Changed too.